platform/android/app/src/main/res/values-de/strings.xml

```diff
@@ -50,7 +50,8 @@
 
     <string name="message_feature_error">Leider gab es einen Fehler in \'%1$s\':\n\n%2$s\n\nIch beende es.</string>
 
-    <string name="button_feature_error_close">OK</string>
+    <string name="button_feature_allow_report">Fehlerbericht an polyPoly senden</string>
```

Comment:
Branding police advisory: `polyPoly` -> `polypoly` (same in English, same on iOS)